src/referencedata-facility/facility.service.js

```diff
@@ -304,10 +303,17 @@
              * @description
              * Retrieves all facilities with id and name fields.
              *
+             * @param  {Object}  paginationParams the pagination params: page, size, sort
              * @return {Promise} Array of facilities with minimal representation
              */
-            function getAllMinimal() {
-                return resource.getAllMinimal().$promise;
+            function getAllMinimal(paginationParams) {
+                var params = (paginationParams !== undefined) ? paginationParams : {
```

Comment:
you don't need to check if it is undefined here, something like if (paginationParams) { would be enough, but I would rather check if sort parameter is set in paginationParamters